### ISSUE TITLE
Basic Appender Format Issue on Node 12

### ISSUE THREAD

**Issue by jd4ever** [log4js-node/log4js-node#869]
```
var log4js = require('log4js');
var logger = log4js.getLogger();
logger.level = 'debug';
logger.debug("Some debug messages");
```

On Node 12, the test snipped above produces the following output
`[2019-05-03T22:08:20.406] [Level { level: 10000, levelStr: 'DEBUG', colour: 'cyan' }] default - Some debug messages`

It should have been
`[2019-05-03T22:08:20.406] [DEBUG] default - Some debug messages`

The incorrect output has only started to appear on Node 12. Node 10 and 11 produced the correct output.

**Comment by Simmyoungsub** [comment]
In my local Node 10 and 12, the output is the same.
I am using version 4.1.0.
What version of the library are you using?

![log4js-node-issue869](https://user-images.githubusercontent.com/18359635/57173188-c6a60780-6e66-11e9-8b7a-9615530d1eca.png)

**Comment by Simmyoungsub** [comment]
#859 fixed

**Comment by nomiddlename** [comment]
Fix published to NPM in log4js@4.1.1
